terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/validation/bytemax.jsp

```diff
@@ -0,0 +1,20 @@
+<h2>@ByteMax Test</h2>

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...